sdk-springboot/pom.xml

```diff
@@ -10,15 +10,27 @@
     <version>1.12.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>dapr-sdk-springboot</artifactId>
-  <packaging>jar</packaging>
+  <groupId>io.dapr.spring</groupId>
+  <artifactId>dapr-sdk-spring-parent</artifactId>
+  <packaging>pom</packaging>
   <version>1.12.0-SNAPSHOT</version>
-  <name>dapr-sdk-springboot</name>
-  <description>SDK extension for Springboot</description>
+  <name>dapr-sdk-spring-parent</name>
+  <description>SDK extension for Spring and Spring Boot</description>
+
+  <modules>
+    <module>sdk-spring-core</module>
+    <module>sdk-springboot-autoconfigure</module>
+    <module>sdk-springboot-starters/sdk-springboot-starter</module>
+    <module>sdk-springboot-testcontainers</module>
+  </modules>
 
   <properties>
-    <maven.deploy.skip>false</maven.deploy.skip>
     <springboot.version>3.2.6</springboot.version>
+    <testcontainers-dapr.version>0.10.14</testcontainers-dapr.version>
```

Comment:
@artur-ciocanu We will not need this one when the PR is merged.

Reply:
True, I will remove once Testcontainers PR is in.